src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,101 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String DELIMITER = ",";
+    private static final int FIELDS = 2;
+    private String[] key = new String[FIELDS];
+    private Integer[] value = new Integer[FIELDS];
+    private int pos = 0;
+    
     public void getStatistic(String fromFileName, String toFileName) {
+        try (BufferedReader bri = new BufferedReader(new FileReader(fromFileName))) {
+            while (bri.ready()) {
+                String line = bri.readLine();
+                applyInput(line);
+            }
+            bri.close();
+        } catch (IOException e) {
+            throw new RuntimeException(e.getMessage() + "Problem with input from CSV file");
+        } catch (NumberFormatException e) {
+            throw new RuntimeException(e.getMessage() + "Numeric parse exception");
+        }
+        try (BufferedWriter bro = new BufferedWriter(new FileWriter(toFileName))) {
+            String result = applyOutput();
+            bro.write(result);
+            bro.close();
+        } catch (IOException e) {
+            throw new RuntimeException(e.getMessage() + "Problem with output to CSV file");
+        }
+    }
+
+    private void applyInput(String line) {
+        String[] em = line.split(DELIMITER);
+        if (em.length == FIELDS) {
+            var tmp1 = em[0].trim();
+            var tmp2 = em[1].trim();
```

Comment:
Introduce `0` and `1` as constants or enum

Reply:
done